### DISCUSSION TITLE
How to go back using Vim Tab

### DISCUSSION THREAD

**Discussion by PraveenKumarpkm** [gdh1995/vimium-c#829] [Q&A] [answered]
I searched for something on google and clicked on a link then, I want to go back to the search engine which can be done by (the **Alt+left arrow** key). 
Is there a shortcut in vim for this?

**Comment by gdh1995** [comment, marked as answer]
There's a command of Vimium C named "`goBack`", and its default key binding is `H` (<kbd>Shift</kbd> and the <kbd>h</kbd> key).

![image](https://user-images.githubusercontent.com/5547703/209476299-f36968eb-f4af-4156-9ad3-cb8dc6658e01.png)

**Reply by PraveenKumarpkm** [reply]
> There's a command of Vimium C named "`goBack`", and its default key binding is `H` (<kbd>Shift</kbd> and the <kbd>h</kbd> key).
> 
> ![image](https://user-images.githubusercontent.com/5547703/209476299-f36968eb-f4af-4156-9ad3-cb8dc6658e01.png)
> 

Hello what about forward, thanks for replying.

Any shortcut for forward

Like after going back with shift+h then, how can I go again?

**Reply by PraveenKumarpkm** [reply]
Oh nevermind, I just read that it's J
Thanks
